### DISCUSSION TITLE
Blobs Azure Download

### DISCUSSION THREAD

**Discussion by arturn1** [balta-io/forum#28] [Tira Dúvidas] [answered]
Qual a melhor abordagem para acessar(downlaod) Blobs, guardar o link em um DB externo e fazer o acesso direto pela URL, ou acessar via container a partir do PATH na Controller?!
Tendo em vista que os arquivos são privates, mas podemos alterar para Only Read.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @arturn1, como vai?

Não sei se entendi direito sua pergunta... o que seria "acessar via container a partir do PATH na Controller"?
Consegue me descrever melhor?

Em geral, como comentei no curso, banco é um recurso caro, então normalmente armazenamos as imagens no Storage e a URL delas em um banco.

Sobre o arquivo ser privado, sem problemas, você pode gerar um Token de acesso que expira em X tempo e serve apenas para aquele download.
